CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ###v0.0.3
 #### Updated
-- Organized library details into tabs
+- Made library details more readable by implementing an accessible Tabs component
```

Comment:
You haven't published `0.0.2` yet so technically you can combined this PR with `0.0.2` if you want to.